Notifee doesn't build with Android gradle plugin v4+, cannot resolve BuildConfig.VERSION_NAME

I'm trying to use Android gradle plugin 4.1.0-rc01 to generate debug symbols to deploy to play store. When using this version of the gradle plugin Notifee doesn't build.

build.gradle
```
classpath 'com.android.tools.build:gradle:4.1.0-rc01'
```

I have little experience with the Android build system so I apologize if this is something obvious I'm doing wrong.

```
NotifeeInitProvider.java:19: error: cannot find symbol
    configBuilder.setProductVersion(BuildConfig.VERSION_NAME);
                                               ^
  symbol:   variable VERSION_NAME
  location: class BuildConfig
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':notifee_react-native:compileDebugJavaWithJavac'.
```

--- a/android/src/main/java/io/invertase/notifee/NotifeeInitProvider.java
+++ b/android/src/main/java/io/invertase/notifee/NotifeeInitProvider.java
@@ -4,6 +4,10 @@
 
 package io.invertase.notifee;
 
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+
 import app.notifee.core.InitProvider;
 import app.notifee.core.Notifee;
 import app.notifee.core.NotifeeConfig;
@@ -16,12 +20,28 @@ public class NotifeeInitProvider extends InitProvider {
     boolean onCreate = super.onCreate();
 
     NotifeeConfig.Builder configBuilder = new NotifeeConfig.Builder();
-    configBuilder.setProductVersion(BuildConfig.VERSION_NAME);
+    configBuilder.setProductVersion(getApplicationVersionString());
     configBuilder.setFrameworkVersion(getReactNativeVersionString());
     configBuilder.setEventSubscriber(new NotifeeEventSubscriber());
 
     Notifee.configure(configBuilder.build());
     return onCreate;
+  }
+
+  private String getApplicationVersionString() {
+    String version = "unknown";
+    Context context = this.getContext();
+    if (context != null) {
+      PackageManager pm = context.getPackageManager();
+      try {
+        PackageInfo pInfo = pm.getPackageInfo(context.getPackageName(), 0);
+        version = pInfo.versionName;
+      } catch (Exception e) {
+        // is there anything useful to log the unbelievably unexpected inability to get package info?
+      }
+    }
+
+    return version;
   }
 
   private String getReactNativeVersionString() {

